classes/autoloader.php

```diff
@@ -41,7 +41,17 @@ public function __construct(array $namespaces = array(), array $namespaceAliases
 			$this->addNamespaceAlias($alias, $target);
 		}
 
-		foreach ($classAliases ?: array('atoum' => __NAMESPACE__ . '\test', __NAMESPACE__ => __NAMESPACE__ . '\test') as $alias => $target)
+		$defaultAliases = array(
+			'atoum' => __NAMESPACE__ . '\test',
+			__NAMESPACE__ => __NAMESPACE__ . '\test'
+		);
+
+		// Remove BC break for PHP_VERSION lower than 7.1.0 void is configured to be an alias of blank.
+		if (version_compare(PHP_VERSION, '7.1.0alpha2') < 0) {
```

Comment:
my only fear here is when conditionally adding the alias we introduce some kind of BC break.

Imagine the following : I coded a custom report for atoum and I used the old `void` class. Everything will work fine on PHP < `7.1`. But when bumping my runtime, I will end up with an error. 

This can't be avoided because we can't `extends` a class conditionally so I'll have to publish two version of my report, one for PHP `< 7.1` and another for `>= 7.1`.

The only valid thing we can do here is just drop the old `void` class and rename it right away to `blank` but this is clearly a BC break in atoum and we must bump to `3.x`.

Bumping atoum means reviewing **every** published extension to adapt the version constraint in the `composer.json` which is a lot of work.

So the only valid patch is, to me, the one were we rename the old class and do not provide an alias. Then we will bump atoum to `3.0`. But I don't want to start a `3.0` without a tiny roadmap for this version.

Moreover, I think before bumping to 3.0, we should merge everything that is ready for 2.x so that our users will have the most up-to-date features in their projects without needing to bump to a new major.

What do you think @atoum/core @atoum/contributors